packages/components/src/components/positions-drawer-card/positions-drawer-card.jsx

```diff
@@ -53,8 +53,8 @@ const PositionsDrawerCard = ({
     const has_ended = !!getEndTime(contract_info);
     const is_mobile = isMobile();
     const contract_card_classname = classNames('dc-contract-card', {
-        'dc-contract-card--green': !is_accumulator && !is_multiplier && profit_loss > 0 && !result,
-        'dc-contract-card--red': !is_accumulator && !is_multiplier && profit_loss < 0 && !result,
+        'dc-contract-card--green': profit_loss > 0 && !result,
```

Comment:
why did we remote these checks?

Reply:
@henry-deriv because in new design gradient should be applied to accumulators and multipliers too in order to stick to the consistency in platform design.